wger/core/api/views.py

```diff
@@ -29,14 +32,82 @@
     WeightUnit)
 from wger.core.api.serializers import (
     UsernameSerializer,
+    UserRegistrationSerializer,
     LanguageSerializer,
     DaysOfWeekSerializer,
     LicenseSerializer,
     RepetitionUnitSerializer,
     WeightUnitSerializer
 )
 from wger.core.api.serializers import UserprofileSerializer
+from wger.core.models import Language
+from rest_framework import status
+from rest_framework.authentication import SessionAuthentication, BasicAuthentication, \
+    TokenAuthentication
 from wger.utils.permissions import UpdateOnlyPermission, WgerPermission
+from django.utils import translation
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from django.contrib.auth.hashers import make_password
+from rest_framework import serializers
+from wger.config.models import GymConfig
+from wger.gym.models import Gym
+from wger.gym.models import GymUserConfig
```

Comment:
This is more of a nitpick, but the imports can be arranged better here -- system -> package -> user